src/pages/EmployeeDetails/EmployeeDetails.tsx

```diff
@@ -0,0 +1,80 @@
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { Input } from '../../shared/UI-components/Input';
+import { useParams } from 'react-router-dom';
+
+const Section = styled.section`
+  padding: 20px;
+  background-color: #f0f0f0;
+`;
+const EmployeeDetailsContainer = styled.div`
+  background: #d5d5d5;
+  width: fit-content;
+  margin: auto;
+  padding: 15px;
+`;
+
+export interface Employee {
+  id: string;
+  name: string;
+  lastName: string;
+  birthDate: string;
+  street: string;
+  city: string;
+  postalCode: string;
+  salary: number;
+  status: 'Employed' | 'On Leave' | 'Fired';
+  phoneNumber: string;
+}
+
+export const EmployeeDetails: React.FC = () => {
+  const { id } = useParams<{ id: string }>();
+  const [employeeData, setEmployeeData] = useState<Employee>();
+  const [error, setError] = useState<string | null>(null);
```

Comment:
Either use error to show something in JSX, or delete this state.

Reply:
Done.